src/transformers/models/deberta/modeling_deberta.py

```diff
@@ -546,17 +546,17 @@ def build_relative_position(query_size, key_size, device):
     return rel_pos_ids
 
 
-@torch.jit.script
```

Comment:
@sgugger  
Obviously I think this is here for a reason, but it is also "blocking" the tracing with torch.fx, quick question: why is it needed?

Reply:
You would have to ask @BigBird01 as it's part of the original implementation. :-)